### PR TITLE
Verify email

### PR DESCRIPTION
**Pull request by sylmak**
Add views for verifying the email and recovering the email. Small refactor of the method to update the user first/last name.

For reviews, this should be compared against reset-pass.

### CODE REVIEW

File: src/components/AccountManagement.vue [hunk 1]
```diff
@@ -70,31 +64,16 @@ async function updateName() {
...
+    await updateUserName(
+      auth.currentUser!,
+      db,
+      firstName.value,
+      lastName.value,
+    );
   } catch (error) {
     // TODO: Show errors in UI.
     console.log(error);
-    return;
```
Comment (skeshive) [inline src/components/AccountManagement.vue]: was this removed intentionally?

File: src/components/RecoverEmailCompleted.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const props = defineProps(['email']);
+
+function goToResetPassword() {
+  router.push({ path: 'reset-password' });
+}
+</script>
+
+<template>
+  <h2>Email recovered</h2>
```
Comment (skeshive) [inline src/components/RecoverEmailCompleted.vue]: hmm why is this called email recovered? is this page reached from the original email? do we undo the email change?

Comment (sylmak) [reply src/components/RecoverEmailCompleted.vue]: Yes, this undoes the email changed. When a user changes their email, they get two emails, one to the old email (to recover) and one to the new email (to verify). If they visit the link to recover, they'll see this. The action mode is called "recoverEmail" by Firebase. I've changed this to "Email restored" if that makes it clearer.

Comment (sylmak) [reply src/components/RecoverEmailCompleted.vue]: Note that this view doesn't actually perform any actions - it's just a notification message. The actual API call to perform email recovery happens in HandleAction and the oobCode must be valid.

Comment (skeshive) [reply src/components/RecoverEmailCompleted.vue]: ah okay I think the text below is a bit confusing. I would reword to "If you didn't initiate the original email change...

**Review by skeshive — APPROVED** [review]
